public/js/helpers/humanize_helper.js

```diff
@@ -158,10 +158,10 @@ var humanize = {
     }
     return Math.floor(seconds) + 's'
   },
-  date: function (stamp, withTimezone) {
+  date: function (stamp, withTimezone, hideHisForMidnight) {
```

Comment:
I like the idea of hiding the time when the stamp is midnight because that is probably just representing the entire day, but I'm confused by the "i" in "His"/"h:i:s". Isn't that hours:minutes:seconds?

Reply:
Yes, it is hours:minutes:seconds
`i` is usually used for minute as `m` has already been used for month in `Y-m-d H:i:s`

Reply:
I don't see that often (php?), but that's OK.  The date formatting I'm used to with nix dates is capital "M" for minutes.  In any case, `hideHisForMidnight` is just a variable name so it might as well have clear meaning.  "His" is a head-scratcher IMO.  The variable could just as well be `hideTimeAtMidnight`.